### PR TITLE
[terra-action-header] Change header-icon CSS Modifiers

### PR DESCRIPTION
**Pull request by amichaelparker**
### Summary
This changes the header-icon CSS modifiers (back, minimize, etc) to use background-image instead of background, so that the modifiers don't overwrite other background properties.

Closes [#2426](https://github.com/cerner/terra-core/issues/2426)

### CODE REVIEW

File: packages/terra-action-header/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Unreleased
 * Updated misleading `children` documentation
 * Made required updates to consume terra-toolkit v5 and terra-dev-site v5
 * Adjusted header-icon top styling
+* Changed header-icon css modifiers (back, minimize, etc) to use background-image
```
Comment (emilyrohrbough) [inline packages/terra-action-header/CHANGELOG.md]: We should add a removed & added section for the changed themeable variables

Comment (amichaelparker) [reply packages/terra-action-header/CHANGELOG.md]: I think I covered this [here](https://github.com/cerner/terra-core/pull/2446/commits/089bba3f41284538b634d4cdf7f35481959c2e11), but let me know if it needs adjustment!

File: packages/terra-action-header/CHANGELOG.md [hunk 2]
```diff
@@ -11,14 +11,19 @@ Unreleased
 
 2.15.0 - (June 11, 2019)
 ------------------
+### Added
+* Added CSS background-image variables in .header-icon for back, close, maximize, minimize, previous, and next
```
Comment (emilyrohrbough) [inline packages/terra-action-header/CHANGELOG.md]: Can we add each variable added and remove so people working on themes knows what changed without looking at the merged PR?

Comment (amichaelparker) [reply packages/terra-action-header/CHANGELOG.md]: Like this? [update](https://github.com/cerner/terra-core/pull/2446/files#diff-b2e7d07bc822cd35b81645ab09c4bdd6)

**Comment by bjankord** [comment]
We want to get PRs out to the theme repos that use these variables before we update and release this.

**Comment by bjankord** [comment]
Talked with the team working on the low-light theme and they are not planning to update their theme repo until July. We'll want to hold onto this change until the theme repos have updated.

**Comment by bjankord** [comment]
We've sent out code and merged it into our theme repos to passively address this change. This is ready to merge now.

**Pull request merged**